ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/OperatingSystem.py

```diff
@@ -97,7 +97,7 @@ def process(self, device, results, log):
             pass
 
         # if NTDS service present then this is a DC
-        if domainController:
+        if len(domainController):
```

Comment:
Technically don't really need to change this..  if domainController would do the same thing, i believe.